lib/workers/pr/changelog/release-notes.ts

```diff
@@ -176,13 +213,21 @@ export async function getReleaseNotesMd(
         `Multiple candidates for changelog file, using ${changelogFile}`
       );
     }
-    const fileRes = await http.getJson<{ content: string }>(
-      `${apiPrefix}/${changelogFile}`
-    );
-    changelogMd =
-      Buffer.from(fileRes.body.content, 'base64').toString() + '\n#\n##';
+    if (apiBaseUrl.includes('gitlab')) {
+      const fileRes = await http.getJson<{ content: string }>(
```

Comment:
```suggestion
      const fileRes = await glGot<{ content: string }>(
```

We need to call gitlab api because of credentials transparently passed by our `got` layer